components/select/style/index.less

```diff
@@ -335,7 +335,7 @@
       right: 4px;
       padding: 0 0 0 8px;
       &:hover {
-        color: #404040;
+        color: @select-multiple-btn-remove-hover;
```

Comment:
Perhaps instead of this, instead do what is done for tag hovers (change opacity)